lint-workflow/lint.py

```diff
@@ -25,9 +25,16 @@ def lint(filename):
             for job_key in jobs:
                 job = jobs[job_key]
 
+                # Make sure runner is using pinned version.
+                runner = job['runs-on']
+                if runner.split("-")[1] == "latest":
+                    findings.append(
+                        f"- Runner version is set to '{runner}', but needs to be pinned to a version.")
+
                 # Check for 'name' key for job.
                 if "name" not in job:
-                    findings.append(f"- Name key missing for job key '{job_key}'.")
+                    findings.append(
```

Comment:
Was this done by `black`?

Reply:
It was done by `autopep8`.  I just changed my formatter to `black` and updated the file.